src/WebJobs.Script.NuGet/WebJobs.Script.nuspec

```diff
@@ -16,9 +16,9 @@
     <dependencies>
       <dependency id="Microsoft.Azure.WebJobs" version="1.2.0-alpha-10306" />
       <dependency id="Microsoft.Azure.WebJobs.ServiceBus" version="1.2.0-alpha-10306" />
-      <dependency id="Microsoft.Azure.WebJobs.Extensions" version="1.1.0-alpha-10287" />
+      <dependency id="Microsoft.Azure.WebJobs.Extensions" version="1.1.0-alpha-10292" />
       <dependency id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="1.1.0-alpha-10287" />
-      <dependency id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10287" />
+      <dependency id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10292" />
       <dependency id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.0.0-alpha-10287" />
```

Comment:
Please move all the extension packages to the 10292 version so they're consistent